pkg/storage/replica.go

```diff
@@ -3297,6 +3327,7 @@ func (r *Replica) unquiesceLocked() {
 			log.Infof(ctx, "unquiescing")
 		}
 		r.mu.quiescent = false
+		r.setLastUpdateTimesLocked()
```

Comment:
Now when the leader becomes from `quiescent` to `unquiescent`, it will update each follower's lastUpdateTime.